helm/cluster-api-provider-azure/values.yaml

```diff
@@ -0,0 +1,19 @@
+name: cluster-api-core
```

Comment:
This doesn't look right, although I think it's not used so we could even remove it.

Reply:
Copy-paste typo, fixed it :grimacing: Not sure if this is used somewhere.